src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHetUnitTest.java

```diff
@@ -182,6 +182,9 @@ public void testAllHetsForLargeCohorts() {
         final double hetsValue = new ExcessHet().calculateEH(allHet, allHet.getGenotypes()).getValue();
 
         Assert.assertTrue(Math.abs(singletonValue) < Math.abs(hetsValue), String.format("singleton=%f allHets=%f", singletonValue, hetsValue));
+
+        //Since all hets is such an extreme case and the sample size is large here, we know that the p-value should be 0
+        Assert.assertTrue(hetsValue == ExcessHet.getMinPhredScaledValue(), String.format("P-value of 0 should be phred scaled to 160.0"));
```

Comment:
one more thing - comparing `doubles` by `==` is always a bad idea. Compare by `assertEquals` and provide a diff threashold